packages/test-utils/src/ConditionalTest.ts

```diff
@@ -46,3 +46,9 @@ export function onNodeVersions(
     });
   }
 }
+
+export function onNotJestJasmine(testBody: () => void): void {
```

Comment:
Perhaps `skipTestOnJasmine` would be better name in this case?

```suggestion
export function skipTestOnJasmine(testBody: () => void): void {
```

Reply:
Thank you for the suggestion, I have updated the function name to 'skipTestOnJasmine'